PROTEUS.env

```diff
@@ -18,8 +18,9 @@ export PATH=$COUPLER_DIR:$PATH
 export PYTHONPATH=$COUPLER_DIR:$PYTHONPATH
 
 # Source SOCRATES if found in SOCRATES directory
-if [ "$(ls -A $COUPLER_DIR/SOCRATES)" ]; then
-    cd $COUPLER_DIR/SOCRATES
+SOC_DIR="$COUPLER_DIR/SOCRATES/"
```

Comment:
Should we replace all references to "coupler" with "proteus" to make it all consistent? Perhaps in a later commit.

Reply:
Yeah that's a good idea. In practice this might be a simple find+replace job, but it should maybe be left for a future commit.